examples/sht3x/README.md

```diff
@@ -0,0 +1,48 @@
+# SHT3x Driver Examples
+
+These examples references two addtional drivers [i2c](https://github.com/kanflo/esp-open-rtos-driver-i2c) and [sht3x](https://github.com/gschorcht/esp-open-rtos/extras/sht3x), which are provided in the `../../extras` folder.
+
```

Comment:
These referenced seem more confusing than useful. You are welcome to include some acknowledgement, by if that is the purpose then I would just do that rather than referencing repos like this.

Reply:
Removed it.

Reply:
@gschorcht, you just experienced one of those somewhat frustrating open source moments when one tries to follow practice already in place only to find that it made people realise the original practice was not a good one. Happens to all of us ;)